### PR TITLE
header changes

### PR DESCRIPTION
**Pull request by vivektripathi2020**
code for the changes of header

### CODE REVIEW

File: source/_patterns/05-pages/journal-article-page.yml [hunk 1]
```diff
@@ -274,19 +275,28 @@ journals_sidebar:
...
-      text: Dynamic Assessment and Computerized Adaptive Tests in Reading Processes
+      title: Computerized Dynamic Testing A Study of the Potential of an Approach Using Sensor Technology
+      text: Computerized Dynamic Testing A Study of the Potential of an Approach Using Sensor Technology
       class: pb-3 border-bottom d-block
 - heading:
     level: '5'
     class: mt-5 pt-3 mb-0 pb-2 border-bottom
-    text: Article keywords
+    text: Keywords
   ullisting_class: list-unstyled pt-3 pb-3 border-bottom 1-lh
```
Comment (vinothk-hw) [inline source/_patterns/05-pages/journal-article-page.yml:300]: please make single BEM approach

File: source/_patterns/05-pages/journal-article-page.yml [hunk 2]
```diff
@@ -1,4 +1,5 @@
 ---
+page_name: journal-article
 show_search: true
 tabcontent_class: border-bottom pt-4 pb-4 mb-5
```
Comment (vinothk-hw) [inline source/_patterns/05-pages/journal-article-page.yml:4]: Please use BEM class

**Review by vinothk-hw — CHANGES_REQUESTED** [review]

File: source/_patterns/05-pages/journal-article-page.yml (hunk 1 above)
Comment (vivektripathi2020) [reply source/_patterns/05-pages/journal-article-page.yml:300]: i will be working on article page changes in new branch

File: source/_patterns/05-pages/journal-article-page.yml (hunk 2 above)
Comment (vivektripathi2020) [reply source/_patterns/05-pages/journal-article-page.yml:4]: i will be working on article page changes in new branch

File: source/_patterns/02-molecules/dropdown/_dropdown.scss [hunk 3]
```diff
@@ -7,7 +7,15 @@
...
   white-space: nowrap;
-  @include caret();
+  // @include caret();  
+}
+.menu_arrow{    
+  @include caret(); 
+}
+.navigation__item__dropdown{
+  .navigation__item__toggle{
+    display: inline-block;
```
[the remark this answers is not in the document]
Comment (vivektripathi2020) [reply source/_patterns/02-molecules/dropdown/_dropdown.scss]: changes done

File: source/_patterns/02-molecules/global/_navigation.scss [hunk 4]
```diff
@@ -146,15 +146,24 @@
...
     &:first-child {
     @extend .ms-desktop-2;
-    @extend .ms-large-desktop-2;
-    }
+    @extend .mx-2;
+    }  
   }
   &__toggle {
     @extend .nav-link;
+    display: inline-block;
```
[the remark this answers is not in the document]
Comment (vivektripathi2020) [reply source/_patterns/02-molecules/global/_navigation.scss]: changes done

File: source/_patterns/02-molecules/global/_navigation.scss [hunk 5]
```diff
@@ -146,15 +146,24 @@
...
+    @include media-breakpoint-down(large-tablet) {
+      width: calc(100% - 16px);
+    }
+  }
+  & .menu_arrow{    
+    @include caret(); 
+    color: $info-light;
+    cursor: pointer;
```
[the remark this answers is not in the document]
Comment (vivektripathi2020) [reply source/_patterns/02-molecules/global/_navigation.scss]: changes done

**Review by vinothk-hw — APPROVED** [review]
Looks Good...

**Pull request merged**